src/app/components/link-custom/link-custom.tsx

```diff
@@ -0,0 +1,54 @@
+import classNames from 'classnames';
+import { Link } from 'react-router-dom';
+import { LinkArrowIcon } from '@/icons';
+
+import styles from './link-custom.module.scss';
+
+type LinkCustomProps = React.DetailedHTMLProps<
+  React.AnchorHTMLAttributes<HTMLAnchorElement>,
+  HTMLAnchorElement
+> & {
+  label: string;
+  href: string;
+  size?: 'small' | 'medium' | 'large';
+  outlined?: boolean;
+  rounded?: boolean;
+  arrow?: boolean;
+  arrowSize?: number;
+  arrowColor?: 'black' | 'white';
+  textRegular?: boolean;
+  className?: string;
+};
+
+export const LinkCustom = ({
+  label,
+  href,
+  size = 'large',
+  outlined = false,
+  rounded = false,
+  arrow = true,
+  arrowSize = 24,
+  arrowColor = 'white',
+  textRegular = false,
+  className = '',
+  ...props
+}: LinkCustomProps) => {
+  const btnClass = classNames(styles.button, styles[size], `stroke:${arrowColor}`, {
+    [styles.outlined]: outlined,
+    [styles.colored]: !outlined,
```

Comment:
I suppose, we don't need two params for this because only one of them can be `true` in your case.

Reply:
Left one on the condition

Reply:
Now it looks like this
![image](https://github.com/rolling-scopes/site/assets/79564962/1e240050-c5d6-4093-a988-69c57846b254)

Reply:
`outlined ? outlined : colored`

Reply:
I did that, but [Quiddlee](https://github.com/Quiddlee) suggested the opposite changes

![Screenshot 2024-05-27 135206](https://github.com/rolling-scopes/site/assets/79564962/42252dee-c9e1-4efa-93ad-872f031f20dd)

Reply:
Changed by `[variant]`